js/tab.js

```diff
@@ -46,7 +46,7 @@ define([
   };
   
   //hopefully this never screws up unaugmented Ace sessions.
-  Tab.prototype = EditSession.prototype;
+  Object.assign(Tab.prototype, EditSession.prototype);
```

Comment:
Can we get away with using `Object.create` instead? I already felt bad enough about sharing a prototype directly.